packages/react-reconciler/src/ReactFiberUnwindWork.js

```diff
@@ -197,6 +240,8 @@ function throwException(
             earliestTimeoutMs = timeoutPropMs;
           }
         }
+      } else if (workInProgress.tag === DehydratedSuspenseComponent) {
+        // TODO
```

Comment:
@acdlite I'm not sure exactly what I need to do here if anything. The timeout doesn't really come into play since failure to hydrate a suspended boundary doesn't timeout until it gets upgraded.

Reply:
I don't think you need to do anything in this branch, but maybe add a comment explaining why: a dehydrated boundary should never time out. The only way the boundary would time out is if it's upgraded to a normal Suspense boundary.